script/upgrades/MU01/governance/MU01-CGP-Phase2.sol

```diff
@@ -238,6 +256,153 @@ contract MU01_CGP_Phase2 is ICGPBuilder, GovernanceScript {
+          cBRLUSDCConfig.referenceRateFeedID,
+          Arrays.addresses(cUSDUSDCConfig.referenceRateFeedID)
+        )
+      )
+    );
+
+    // Enable Median Delta Breaker for rate feeds
+    for (uint256 i = 0; i < poolConfigs.length; i++) {
+      if (poolConfigs[i].isMedianDeltaBreakerEnabled) {
+        transactions.push(
+          ICeloGovernance.Transaction(
+            0,
+            breakerBox,
+            abi.encodeWithSelector(
+              BreakerBox(0).toggleBreaker.selector,
+              medianDeltaBreaker,
+              poolConfigs[i].referenceRateFeedID,
+              true
+            )
+          )
+        );
+      }
+    }
+
+    // Enable Value Delta Breaker for rate feeds
+    for (uint256 i = 0; i < poolConfigs.length; i++) {
+      if (poolConfigs[i].isValueDeltaBreakerEnabled) {
+        transactions.push(
+          ICeloGovernance.Transaction(
+            0,
+            breakerBox,
+            abi.encodeWithSelector(
+              BreakerBox(0).toggleBreaker.selector,
+              valueDeltaBreaker,
+              poolConfigs[i].referenceRateFeedID,
+              true
+            )
+          )
+        );
+      }
+    }
+
+    // Set BreakerBox address in SortedOracles
+    transactions.push(
+      ICeloGovernance.Transaction(
+        0,
+        contracts.celoRegistry("SortedOracles"),
+        abi.encodeWithSelector(SortedOracles(0).setBreakerBox.selector, breakerBox)
+      )
+    );
+  }
+
+  function proposal_configureMedianDeltaBreaker() public {
+    // Set the cooldown times
+    transactions.push(
+      ICeloGovernance.Transaction(
+        0,
+        medianDeltaBreaker,
+        abi.encodeWithSelector(
+          MedianDeltaBreaker(0).setCooldownTime.selector,
+          Arrays.addresses(cUSD, cEUR, cBRL),
+          Arrays.uints(
+            cUSDCeloConfig.medianDeltaBreakerCooldown,
+            cEURCeloConfig.medianDeltaBreakerCooldown,
+            cBRLCeloConfig.medianDeltaBreakerCooldown
+          )
+        )
+      )
+    );
+
+    // Set the rate change thresholds
+    transactions.push(
+      ICeloGovernance.Transaction(
+        0,
+        medianDeltaBreaker,
+        abi.encodeWithSelector(
+          MedianDeltaBreaker(0).setRateChangeThresholds.selector,
+          Arrays.addresses(cUSD, cEUR, cBRL),
+          Arrays.uints(
+            cUSDCeloConfig.medianDeltaBreakerThreshold.unwrap(),
+            cEURCeloConfig.medianDeltaBreakerThreshold.unwrap(),
+            cBRLCeloConfig.medianDeltaBreakerThreshold.unwrap()
+          )
+        )
+      )
+    );
+
+    // Set smoothing factor for rate feeds
+    for (uint i = 0; i < poolConfigs.length; i++) {
+      if (poolConfigs[i].smoothingFactor != 0)
+        transactions.push(
+          ICeloGovernance.Transaction(
+            0,
+            medianDeltaBreaker,
+            abi.encodeWithSelector(
+              MedianDeltaBreaker(0).setSmoothingFactor.selector,
+              poolConfigs[i],
```

Comment:
i think this should be the rateFeedID and not the whole pool config struct

Reply:
correct